### PR TITLE
Feature/truenth usa footer layout update

### PR DESCRIPTION
**Pull request by achen2401**
address https://jira.movember.com/browse/TN-2250
new layout of footer based on new prototype provided here:
https://marvelapp.com/cfaddch/screen/61502014

Noting some duplications in content between Truenth USA footer and the portal footer html provided by portal API, I have created a story to clean up the code:  https://jira.movember.com/browse/TN-2285

### CODE REVIEW

File: portal/gil/templates/gil/macros.html [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
-        <ul class="nav-list nav-list--lower icon-list">
-          <li class="nav-list__item"><a href="{{url_for('gil.contact')}}" ><i class="fa fa-envelope fa-2x" aria-hidden="true"></i>{{_("Contact Us")}}</a></li>
-          <li class="nav-list__item"><a href="https://www.facebook.com/TrueNTH.ProstateCancer/" target="_blank"><i class="fa fa-facebook-square fa-2x" aria-hidden="true"></i>{{_("Facebook")}}</a></li>
-          <li class="nav-list__item"><a href="https://twitter.com/TrueNTH_US" target="_blank"><i class="fa fa-twitter-square fa-2x" aria-hidden="true"></i>{{_("Twitter")}}</a></li>
-        </ul>
-        <div class="footer__logo-area"><a href="http://www.movember.com" target="_blank">
-            <figure class="footer__logo--truenth"></figure>
-            <figure class="footer__logo--movember"></figure></a>
+        <div class="main-nav-list">
+          <div class="nav-image"><a href="http://www.movember.com" target="_blank"><figure class="footer__logo--truenth"></figure></a></div>
```
Comment (ivan-c) [inline portal/gil/templates/gil/macros.html]:
```suggestion
          <div class="nav-image"><a href="https://www.movember.com" target="_blank"><figure class="footer__logo--truenth"></figure></a></div>
```
https is preferred if it's available, especially if it's on a link to a domain that the user may login on

File: portal/gil/templates/gil/portal.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h4 class="icon-box__lowhead">{{ _("Complete Registration") }}</h4>
     </section>
     <br class="clearfix"/>
     <br/>
-    {{footerHTML("home--footer-container")}}
+    {{footerHTML("home--footer-container", user)}}
```
Comment (ivan-c) [inline portal/gil/templates/gil/portal.html]: keyword arguments can be better for functions with 2 or more args; they're self-documenting and each function call doesn't need to be updated if the function signature changes
```suggestion
    {{footerHTML(custom_class="home--footer-container", user=user)}}
```

File: portal/gil/templates/gil/macros.html [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
+        </div>
+        <hr class="hr">
+        <div class="footer__logo-area">
+          <div class="item links">
+              <span class="nav-list-secondary__item"><a href="{{url_for('gil.terms_and_conditions')}}">{{_("Terms & Conditions")}}</a></span>
+              <span class="nav-list-secondary__item"><a href="{{url_for('gil.privacy')}}">{{_("Privacy Policty")}}</a></span>
+              <div class="copyright">&copy;{{_("%(year)d Movember Foundation. All rights reserved. A registered 501(c)3 non-profit organization.", year=config.COPYRIGHT_YEAR)}}</div>
           </div>
-        <div class="footer__logo-area copyright">&copy;{{_("%(year)d Movember Foundation. All rights reserved. A registered 501(c)3 non-profit organization.", year=config.COPYRIGHT_YEAR)}}</div>
+          <div class="image"><a href="http://www.movember.com" target="_blank"><figure class="footer__logo--movember"></figure></a></div>
```
Comment (ivan-c) [inline portal/gil/templates/gil/macros.html]:
```suggestion
          <div class="image"><a href="https://www.movember.com" target="_blank"><figure class="footer__logo--movember"></figure></a></div>
```

File: portal/templates/portal_footer.html [hunk 4]
```diff
@@ -1,55 +1,64 @@
...
+      </ul>
+    </div>
+    <hr class="hr">
+    <div class="footer__logo-area">
+      <div class="item">
+          <span class="nav-list-secondary__item"><a href="{{PORTAL}}/terms">{{_("Terms")}}</a></span>
+          <span class="nav-list-secondary__item"><a href="{{PORTAL}}/privacy">{{_("Privacy")}}</a></span>
+          <div class="copyright">&copy;{{_("%(year)d Movember Foundation. All rights reserved. A registered 501(c)3 non-profit organization.", year=config.COPYRIGHT_YEAR)}}</div>
+      </div>
+      <div class="image"><a href="http://www.movember.com" target="_blank"><figure class="footer__logo--movember"></figure></a></div>
```
Comment (ivan-c) [inline portal/templates/portal_footer.html]:
```suggestion
      <div class="image"><a href="https://www.movember.com" target="_blank"><figure class="footer__logo--movember"></figure></a></div>
```

**Comment by achen2401** [comment]
thank u, ivan, merged.

**Pull request merged**